Dockerfile

```diff
@@ -1,13 +1,31 @@
-FROM golang:1.21 as builder
-WORKDIR /go/src/app
+FROM golang:1.21 AS builder
+WORKDIR /workspace
 COPY go.mod go.sum ./
 RUN go mod download
-COPY . .
-RUN CGO_ENABLED=0 make build
 
-FROM scratch
+COPY abi/ abi/
```

Comment:
Why did you choose not to use `COPY . .`? Since we're using multistage dockerfile, it's safe since anything that isn't copied in the `gcr.io/distroless/static:nonroot` will be discarded. There's also the option of `.dockerignore` files.

Reply:
100% agree w/ you, I had to remove `COPY . .` because of SonaQube alert :(